File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        arr = [0, 0]
        for i in range(len(nums)):
            answer = target - nums[i]
            for j in range(len(nums)):
                if answer == nums[j] and i != j:
                    return i, j
        return i, j

File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] result = new int[2];
        for(int i = 0; i < nums.length; i++) {
            int value = target - nums[i];
            for(int j = i + 1; j < nums.length; j++) {
                if (value == nums[j]) {
                    return new int[]{i, j};
                }
            }
        }
        return result;
    }
}